### DISCUSSION TITLE
Compatibility for MacBook Pro (14-inch, 2023)

### DISCUSSION THREAD

**Discussion by konjohn96** [DOI-USGS/ISIS3#5318] [Q&A]
Hi, I would like to install and use the latest ISIS but it seems like my old laptop might not be able to support it.
Can anyone let me know if the latest MacBook Pro (14 inch) running on macOS Ventura is able to install and run the software?
Thank you in advance

**Comment by AustinSanders** [comment]
Hey @konjohn96 ,

In terms of processing power, the latest MacBook Pro is certainly a beefy enough machine to use ISIS applications!  One consideration, though, is that many Mac machines are using M1/M2 chipsets -- if I understand correctly, _all_ the latest Macs are on M1/M2 architecture.  We've got some M1 specific installation instructions, but support for this architecture is in its infancy.

You can find a summary of one user's experience performing a dev installation of ISIS on an M1 machine at #5188 .  The [README](https://github.com/DOI-USGS/ISIS3#readme) also includes special instructions for Mac M1 users.
